gateways/paypalpayflow/request_builder.go

```diff
@@ -0,0 +1,51 @@
+package paypalpayflow
+
+import (
+	"fmt"
+
+	"github.com/BoltApp/sleet"
+)
+
+var (
+	defaultVerbosity string = "HIGH"
+	defaultTender    string = "C"
+)
+
+func buildAuthorizeParams(request *sleet.AuthorizationRequest) *Request {
+	expirationDate := fmt.Sprintf("%02d%02d", request.CreditCard.ExpirationMonth, request.CreditCard.ExpirationYear%100)
+	amount := sleet.AmountToDecimalString(&request.Amount)
+	return &Request{
+		TrxType:            "A",
+		Amount:             &amount,
+		CreditCardNumber:   &request.CreditCard.Number,
+		CardExpirationDate: &expirationDate,
+		Verbosity:          &defaultVerbosity,
+		Tender:             &defaultTender,
```

Comment:
let's add the optional billing address fields here, it's included in the authorization request

Reply:
Done